app/models/organization_member.rb

```diff
@@ -3,6 +3,14 @@
 class OrganizationMember < ApplicationRecord
   belongs_to :organization
   has_one :user, as: :referencee, dependent: :destroy, required: false, autosave: true
+  has_one :service_specification_contact_person,
```

Comment:
@lukasbischof shouldn't thid actually be has_many? One OrganizationMember can be added to more then one service specification.

Reply:
How would this then be represented on the print version?

Reply:
The specification still has one lead and one contact person. But this right now limits them to be added to one specification max. Or am I totally confused?